### ISSUE TITLE
ERR_UNSUPPORTED_ESM_URL_SCHEME - received protocol 'd:' [v7]

### ISSUE THREAD

**Issue by fmilioni** [i18next/i18next-parser#698]
## 🐛 Bug Report

CLI command `i18next` throws the following exception:

```bash
node:internal/errors:484
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only URLs with a scheme in: file, data are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'd:'
    at new NodeError (node:internal/errors:393:5)
    at throwIfUnsupportedURLScheme (node:internal/modules/esm/resolve:1026:11)
    at defaultResolve (node:internal/modules/esm/resolve:1106:3)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:841:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ESMLoader.import (node:internal/modules/esm/loader:525:22)
    at importModuleDynamically (node:internal/modules/esm/translators:110:35)
    at importModuleDynamicallyCallback (node:internal/process/esm_loader:35:14)
    at _callee$ (file:///D:/projects/xxxx/node_modules/.pnpm/i18next-parser@7.0.3/node_modules/i18next-parser/dist/helpers.js:268:336) {
  code: 'ERR_UNSUPPORTED_ESM_URL_SCHEME'
}
```

## To Reproduce

- I think it is related to the fact that my project is in partition `D:`
- Must use .js (and maybe .ts) config file (related to esm loader)
- using pnpm as package manager
- Previous version (6.6.0) works fine

## Config file (.js)

```js
module.exports = {
  indentation: 2,
  keepRemoved: false,
  defaultNamespace: 'common',
  namespaceSeparator: false,
  locales: ['en', 'pt', 'vi', 'zh'],
  lineEnding: 'lf',
  input: 'src/**/*.{ts,tsx}',
  output: 'public/locales/$LOCALE/$NAMESPACE.json',
  sort: true,
};

```

## ➡️ Workaround ⬅️

If viable, convert your `.js` config file into a `.json` file format

```json
{
  "indentation": 2,
  "keepRemoved": false,
  "defaultNamespace": "common",
  "namespaceSeparator": false,
  "locales": ["en", "pt", "vi", "zh"],
  "lineEnding": "lf",
  "input": "src/**/*.{ts,tsx}",
  "output": "public/locales/$LOCALE/$NAMESPACE.json",
  "sort": true
}

```

## Environment

- *runtime version*: node v16.17.0 and node v18.12.1 
- using *next-i18next version*: 12.1.0
- *os*: Windows
- *path*: D:\projects\xxxxx\

**Comment by BenasB** [comment]
Facing same problem, converting to json makes using `defaultValue: function` unavailable

**Comment by karellm** [comment]
I have limited bandwidth these days and no access to a windows machine. Can anyone facing this issue submit a PR to fix this?

**Comment by Chriskuiper1** [comment]
I have the same problem, doesn't work.

(node:27224) UnhandledPromiseRejectionWarning: Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only file and data URLs are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'd:'

**Comment by karellm** [comment]
Fixed in `7.1.0`
